etc/saq.default.ini

```diff
@@ -275,6 +275,12 @@ ignore_easy_button = on
 upload_vt = on
 upload_vxstream = on
 view_in_vx = on
+; Allows analysts to clear cached URL content
+clear_cloudphish_alert = on
+
+; Show the total alerts resulting from the current filter.
+;  Ex: "Alert (22)" would say the are 22 alerts matching the filter.
+show_total_alert_count = no
```

Comment:
Any reason we would ever not want to show this?

Reply:
I don't think so.. I was surprised it wasn't on there already. Want me to make it default on or default on and not configurable?

The other route was to remove the code that hides the pagination control at the bottom of the manage page, however, it's at the bottom and I didn't want to scroll to see what it says. I also like it not being there when there is nothing in the Q.